AdyenComponents/UPI/UPIComponent.swift

```diff
@@ -94,7 +94,8 @@ public final class UPIComponent: PaymentComponent,
 
     /// The upi selection segment control item to choose the upi flow.
     internal lazy var upiFlowSelectionItem: FormSegmentedControlItem = {
-        let item = FormSegmentedControlItem(items: ["VPA", "QR code"], style: configuration.style.segmentedControlStyle,
+        let item = FormSegmentedControlItem(items: ["Pay by any UPI app", "Other UPI options"],
```

Comment:
Is this not localized?

Reply:
No, I have created the job on smartling for this but there are some issues happening on smartling so once the job is completed I will replace this strings. FYI, there is a separate task for this in subtasks list :)